Modules/QuestieInit.lua

```diff
@@ -0,0 +1,237 @@
+        end
+    else
+        coroutine.yield()
+        l10n:Initialize()
+
+        coroutine.yield()
+        QuestieCorrections:MinimalInit()
+    end
+
+    if not Questie.db.char.townsfolk then
+        coroutine.yield()
+        QuestieMenu:BuildCharacterTownsfolk()
+    end
+
+    coroutine.yield()
+    QuestieDB:Initialize()
+
+    QuestieLib:CacheAllItemNames() -- todo: remove this, blizzard said we shouldn't query more than once a second
+    QuestieCleanup:Run()
+
+    -- register events that rely on questie being initialized
+    QuestieEventHandler:RegisterLateEvents()
+
+    QuestieMap:InitializeQueue()
+
+    -- hack_prime_log()  -- this seems to make it update the data much quicker
+    coroutine.yield()
+    for i=1, GetNumQuestLogEntries() do
+        GetQuestLogTitle(i)
+        coroutine.yield()
+        QuestieQuest:GetRawLeaderBoardDetails(i)
+    end
+
+    coroutine.yield()
+    QuestiePlayer:Initialize()
+    l10n:PostBoot()
+
+    coroutine.yield()
+    QuestieJourney:Initialize()
+    coroutine.yield()
+    QuestieQuest:Initialize()
+    coroutine.yield()
+    QuestieQuest:GetAllQuestIdsNoObjectives()
+    coroutine.yield()
+    QuestieQuest:CalculateAndDrawAvailableQuestsIterative()
+    coroutine.yield()
+    QuestieNameplate:Initialize()
+    coroutine.yield()
+    QuestieMenu:PopulateTownsfolkPostBoot()
+    Questie:Debug(DEBUG_ELEVATED, "PLAYER_ENTERED_WORLD")
+    didPlayerEnterWorld = true
+
+    -- Initialize the tracker
+    coroutine.yield()
+    QuestieTracker:Initialize()
+
+    local dateToday = date("%y-%m-%d")
+
+    if Questie.db.char.showAQWarEffortQuests and (Questie.db.char.aqWarningPrintDate == nil or Questie.db.char.aqWarningPrintDate < dateToday) then
+        Questie.db.char.aqWarningPrintDate = dateToday
+        C_Timer.After(2, function()
+            print("|cffff0000-----------------------------|r")
+            Questie:Print("|cffff0000The AQ War Effort quests are shown for you. If your server is done you can hide those quests in the General settings of Questie!|r");
+            print("|cffff0000-----------------------------|r")
+        end)
+    end
+
+    QuestieMenu:OnLogin()
+
+    if Questie.db.global.debugEnabled then
+        QuestieLoader:PopulateGlobals()
+    end
+
+    Questie.started = true
+
+end
+
+function QuestieInit:LoadDatabase(key)
+    if QuestieDB[key] then
+        coroutine.yield()
+        QuestieDB[key] = loadstring(QuestieDB[key]) -- load the table from string (returns a function)
+        coroutine.yield()
+        QuestieDB[key] = QuestieDB[key]() -- execute the function (returns the table)
+    else
+        Questie:Debug(DEBUG_DEVELOP, "Database is missing, this is likely do to era vs tbc: ", key)
+    end
+end
+
+function QuestieInit:LoadBaseDB()
+    -- reset townsfolk on all characters before compile
+    for _, char in pairs(QuestieConfig.char) do
+        char.townsfolk = nil
+    end
```

Comment:
Why are we now accessing `QuestieConfig` again? I thought we agreed that it is bad and we just want to use `Questie.db`